x-pack/test/reporting/api/generate/fixtures.ts

```diff
@@ -158,3 +158,27 @@ export const CSV_RESULT_NANOS = `date,message,"_id"
 "2015-01-01T12:10:30.123456789Z","Hello 2",
 "2015-01-01T12:10:30","Hello 1",
 `;
+
```

Comment:
Maybe we could use something like the backtick operator with `dedent`? I've seen that lib used for our dockerfiles and elsewhere. Just a bit more pleasant to work on.

Reply:
export const JOB_PARAMS_IN_QUERYSTRING =
  dedent`(conflictedTypesFields:!(),fields:!(%27@timestamp%27,clientip,extens
  ion),indexPatternId:%27logstash-*%27,metaFields:!(_source,_id,_type,_index,_scor
  e),searchRequest:(body:(_source:(excludes:!(),includes:!(%27@timestamp%27,client
  ip,extension)),docvalue_fields:!(),query:(bool:(filter:!((match_all:()),(range:(
  %27@timestamp%27:(gte:%272015-09-20T10:19:40.307Z%27,lt:%272015-09-20T10:26:56.2
  21Z%27))),(range:(%27@timestamp%27:(format:strict_date_optional_time,gte:%272004
  -09-17T21:19:34.213Z%27,lte:%272019-09-17T21:19:34.213Z%27)))),must:!(),must_not
  :!(),should:!())),script_fields:(),sort:!((%27@timestamp%27:(order:desc,unmapped
  _type:boolean))),stored_fields:!(%27@timestamp%27,clientip,extension),version:!t
  ),index:%27logstash-*%27),title:%27A%20Saved%20Search%20With%20a%20DATE%20FILTER
  %27,type:search)`;

Reply:
I like it! It turns out `dedent` still retains the line breaks, so I added in a short little function to act as the template for the string.